infra/app/envs/dev/main.tf

```diff
@@ -8,14 +8,14 @@ locals {
   # if you choose not to use workspaces set this value to "dev" 
   prefix = terraform.workspace
   # Choose the region where this infrastructure should be deployed.
-  region = "us-west-1"
+  region = "us-west-2"
   # Add environment specific tags
   tags = merge(module.project_config.default_tags, {
     environment = local.environment_name
     description = "Application resources created in dev environment"
   })
 
-  tfstate_bucket = "wic-prp-636249768232-us-west-1-tf-state"
+  tfstate_bucket = "wic-prp-636249768232-us-west-2-tf-state"
```

Comment:
I'm sure these buckets already exist, but if this were a project with more s3 buckets, i'd prefer to have the informational parts of the bucket name (`tf-state`) in the front and the "uniqueness" parts of the bucket name at the end